lib/encryption_algorithm.rb

```diff
@@ -1,14 +1,20 @@
 require 'date'
-require_relative 'shiftable'
-require_relative 'alpha_numable'
+require './lib/shift_gen'
+require './lib/alpha_num'
 
 class EncryptionAlgorithm
-  include Shiftable
-  include AlphaNumable
+  # include Shiftable
 
-  attr_reader :message
+
+  attr_reader :message, :alpha_num, :shift_gen
   def initialize
     @message = []
+    couple
+  end
+
+  def couple
+    @alpha_num = AlphaNum.new
+    @shift_gen = ShiftGen.new
   end
```

Comment:
This is what I am most comfortable with. Passing them in as initialize arguments was my other option. Because enigma doesn't have any initialize arguments I went this direction